.github/scripts/update_homebrew_formula.py

```
@@ -0,0 +1,118 @@
+    "x86_64-apple-darwin",
+    "x86_64-unknown-linux-gnu",
+)
+
+
+def asset_name(tag: str, target: str) -> str:
+    return f"git-smee-{tag}-{target}.tar.gz"
+
+
+def version_from_tag(tag: str) -> str:
+    if not re.fullmatch(r"v[0-9]+(\.[0-9]+)*([-.][0-9A-Za-z.]+)?", tag):
+        raise ValueError(f"invalid release tag: {tag!r}")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)